tests/metrics/classification/test_precision_recall_curve.py

```diff
@@ -4,11 +4,10 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import List, Optional
+from typing import Optional
 
 import torch
 
-from sklearn.metrics import precision_recall_curve
```

Comment:
Thanks for catching this error! Let's have a rebase to make sure we don't have code change in unrelated files.